### PR TITLE
MAP-1462 update realease notes

### PR DESCRIPTION
**Pull request by GurnankCheema**
A change of config to fix a build  error when upgrading a project's gradle-spring-boot plugin to 6.0.0 and above

### CODE REVIEW

File: release-notes/6.x.md [hunk 1]
```diff
@@ -75,6 +75,36 @@ Database support has been moved to separate database packages rather than being
 ```
 Please see [upgrading from Spring Boot 3.2](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.3-Release-Notes#upgrading-from-spring-boot-32) for more information.
 
+## H2 and Hibernate for tests
+Potential errors similar to the following may occur in the build process:
+
+Caused by: org.hibernate.exception.SQLGrammarException: could not prepare statement ...
```
Comment (petergphillips) [inline release-notes/6.x.md:81]: would be better to format these two lines as code, as they are then joined together when viewing the markdown

File: release-notes/6.x.md [hunk 2]
```diff
@@ -75,6 +75,36 @@ Database support has been moved to separate database packages rather than being
...
+Caused by: org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement ...
+
+These may potentially be resolved by ensuring hibernate is configured to know that H2 rather than 
+postgress is used in the test environment
+
+### application.yml
+```kotlin
```
Comment (petergphillips) [inline release-notes/6.x.md:88]: trivial point - code should be of `yaml` type instead I think

File: release-notes/6.x.md [hunk 3]
```diff
@@ -75,6 +75,36 @@ Database support has been moved to separate database packages rather than being
...
+    data:
+        jdbc:
+            dialect: postgresql
+    sql:
+        init:
+            continue-on-error: true
+```
+
+### application-test.yml
```
Comment (petergphillips) [inline release-notes/6.x.md:97]: trivial point - code should be of `yaml` type instead I think

File: release-notes/6.x.md [hunk 4]
```diff
@@ -75,6 +75,36 @@ Database support has been moved to separate database packages rather than being
...
+postgress is used in the test environment
+
+### application.yml
+```kotlin
+    data:
+        jdbc:
+            dialect: postgresql
+    sql:
+        init:
+            continue-on-error: true
```
Comment (petergphillips) [inline release-notes/6.x.md:94]: not sure this `continue-on-error` is relevant here - I think it was probably changed in the PR as the old way was deprecated?

Comment (GurnankCheema) [reply release-notes/6.x.md:94]: in fact don't need to include this at all

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**